.github/workflows/be_user_service.yml

```diff
@@ -43,10 +43,22 @@ jobs:
          architecture: x64
 
       - name: Build with maven
-        run: mvn dependency:tree clean install
+        run: mvn clean install
```

Comment:
The log should be verbose to find conflicts ..

Reply:
added back the verbosecontent during build